### PR TITLE
Use Go 1.8

### LINKED ISSUE #2576: update to go1.8
so we can update weaveworks/common to latest (#2570)

It has not yet been officially released in `weaveworks/scope-backend-build`s base image: https://launchpad.net/ubuntu/yakkety/+source/golang-1.8

### PR DESCRIPTION
**Pull request by 2opremio**
Fixes #2576

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-ENV GOVERSION 1.7
-ENV PATH /go/bin:/usr/lib/go-${GOVERSION}/bin:/usr/bin:/bin:/usr/sbin:/sbin
+ENV GOVERSION 1.8
+ENV PATH /go/bin:/usr/local/go/bin:/usr/bin:/bin:/usr/sbin:/sbin
 ENV SCOPE_SKIP_UI_ASSETS true
 RUN apt-get update && \
-	apt-get install -y libpcap-dev python-requests time file shellcheck golang-${GOVERSION} git gcc-arm-linux-gnueabihf curl && \
+	apt-get install -y libpcap-dev python-requests time file shellcheck git gcc-arm-linux-gnueabihf curl && \
 	rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+RUN curl -s https://storage.googleapis.com/golang/go#{GO_VERSION}.linux-amd64.tar.gz | tar xz -C /usr/local
```
Comment (rndstr) [inline backend/Dockerfile]: `${GO_VERSION}`

**Comment by rndstr** [comment]
nit: fixes #2576 :)

**Comment by 2opremio** [comment]
@rndstr PTAL

**Comment by rndstr** [comment]
awesome! LGTM

**Pull request merged**